sites/next.skeleton.dev/src/content/docs/guides/figma.mdx

```diff
@@ -0,0 +1,27 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+title: Figma UI kit
+description: Welcome to the Skeleton Figma Design System Tutorials!
+order: 60
+---
+
+export const components = componentSet;
+
+import FigmaBuyNow from '@components/docs/FigmaBuyNow.astro';
+import NavGrid from '@components/docs/NavGrid.astro';
+
+import { Image } from 'astro:assets';
+import ImgHero from '@images/get-started/figma/hero.webp';
+import ImgComponents from '@images/get-started/figma/components.webp';
+import ImgPlugin from '@images/get-started/figma/plugin.webp';
+import ImgTutorial from '@images/get-started/figma/tutorials.webp';
+import ImgKit from '@images/get-started/figma/ui-kit.webp';
+import { Clock3 } from 'lucide-react';
+import { Shapes } from 'lucide-react';
+import { SwatchBook } from 'lucide-react';
+
+Explore step-by-step guides to unlock the full potential of Skeleton in your design workflow. Whether you're setting up for the first time or mastering advanced features, these tutorials will guide you every step of the way.
+
+Discover all the benefits of <a href='/figma' className='anchor'>Skeleton Figma UI library. </a>
```

Comment:
The callout is getting lost here. I might suggest you utilize one of these large button panels like shown here:

https://next.skeleton.dev/docs/get-started/plugin#base

Make the CTA something like "Get the Figma UI Kit".

![Screenshot 2024-12-16 at 2 40 55 PM](https://github.com/user-attachments/assets/5eee141a-2dc3-409a-a8da-e8b605001db4)